app/tests/api/scripts/test_create_eligibility_screener_csv.py

```diff
@@ -0,0 +1,161 @@
+        assert csv_record[ELIGIBILITY_SCREENER_CSV_HEADER.last_name] == db_record.last_name
+        assert csv_record[ELIGIBILITY_SCREENER_CSV_HEADER.phone_number] == db_record.phone_number
+        assert csv_record[ELIGIBILITY_SCREENER_CSV_HEADER.eligibility_categories] == join_list(
+            db_record.eligibility_categories
+        )
+        assert csv_record[
+            ELIGIBILITY_SCREENER_CSV_HEADER.has_prior_wic_enrollment
+        ] == blank_for_null(db_record.has_prior_wic_enrollment)
+        assert csv_record[ELIGIBILITY_SCREENER_CSV_HEADER.eligibility_programs] == join_list(
+            db_record.eligibility_programs
+        )
+        assert csv_record[ELIGIBILITY_SCREENER_CSV_HEADER.household_size] == blank_for_null(
+            db_record.household_size
+        )
+        assert csv_record[ELIGIBILITY_SCREENER_CSV_HEADER.zip_code] == db_record.zip_code
+        assert csv_record[ELIGIBILITY_SCREENER_CSV_HEADER.chosen_wic_clinic] == "TODO"
+        assert csv_record[ELIGIBILITY_SCREENER_CSV_HEADER.applicant_notes] == blank_for_null(
+            db_record.applicant_notes
+        )
+        assert csv_record[
+            ELIGIBILITY_SCREENER_CSV_HEADER.submitted_datetime
+        ] == db_record.created_at.strftime(DATETIME_OUTPUT_FORMAT)
+
+        # Verify the eligibility screener record  had its timestamp updated
+        assert db_record.added_to_eligibility_screener_at is not None
+        # and that the timestamp is more than the created_at value (just to show it happened slightly later)
+        assert db_record.added_to_eligibility_screener_at > db_record.created_at
+
+
+def test_create_eligibility_screener_csv(
+    test_db_session, initialize_factories_session, tmp_file_path
+):
+    db_records = []
+    # To make validating these easier in the CSV, make the first names consistent
+    db_records.append(
+        EligibilityScreenerFactory.create(
+            # Routine case
+            first_name="A"
+        )
+    )
+    db_records.append(
+        EligibilityScreenerFactory.create(
+            # Override the factory to flip nullable fields
+            first_name="B",
+            eligibility_categories=None,
+            eligibility_programs=None,
+            household_size=5,
+            applicant_notes=None,
+        )
+    )
+    db_records.append(
+        EligibilityScreenerFactory.create(
+            # Multiple records in arrays
+            first_name="C",
+            eligibility_categories=["baby", "pregnant", "guardian"],
+            eligibility_programs=["tanf", "snap", "insurance"],
+        )
+    )
+
+    create_eligibility_screener_csv(test_db_session, tmp_file_path)
+    csv_rows = read_csv_records(tmp_file_path)
+    validate_csv_records(db_records, csv_rows, test_db_session)
+
+    # If we run it again, just a blank file should be made
+    create_eligibility_screener_csv(test_db_session, tmp_file_path)
+    csv_rows = read_csv_records(tmp_file_path)
+    assert len(csv_rows) == 0
```

Comment:
Why doesn't this return 1 row for the header?

Reply:
Oh got it. It's the difference between `convert_eligibility_screener_records_for_csv()` and `csv.DictReader()`. Thanks!